paseos/tests/communication_test.py

```diff
@@ -0,0 +1,611 @@
+"""Tests to check the communication models creation and calculation function(s)"""
+import numpy as np
+import pykep as pk
+import pytest
+
+from paseos.actors.actor_builder import ActorBuilder
+from paseos.actors.ground_station_actor import GroundstationActor
+from paseos.actors.spacecraft_actor import SpacecraftActor
+from paseos.communication.device_type import DeviceType
+from paseos.communication.optical_link_model import OpticalLinkModel
+from paseos.communication.optical_receiver_model import OpticalReceiverModel
+from paseos.communication.optical_transmitter_model import OpticalTransmitterModel
+from paseos.communication.radio_link_model import RadioLinkModel
+from paseos.communication.radio_receiver_model import RadioReceiverModel
+from paseos.communication.radio_transmitter_model import RadioTransmitterModel
+from paseos.communication.receiver_model import ReceiverModel
+from paseos.communication.transmitter_model import TransmitterModel
+
+
+def test_link_creation():
+    t0 = pk.epoch_from_string("2023-Jan-04 20:00:00")
+    maspalomas_groundstation = ActorBuilder.get_actor_scaffold(
+        name="maspalomas_groundstation", actor_type=GroundstationActor, epoch=t0
+    )
+    receiver_name = "maspalomas_radio_receiver_1"
+
+    ActorBuilder.add_comm_device(
+        actor=maspalomas_groundstation,
+        device_name=receiver_name,
+        noise_temperature=135,
+        line_losses=1,
+        polarization_losses=3,
+        antenna_gain=62.6,
+        device_type=DeviceType.RADIO_RECEIVER,
+    )
```

Comment:
if you have some documentation for these values / how they are chosen would be good to put it here